run.ps1

```diff
@@ -0,0 +1,147 @@
+<#
+.SYNOPSIS
+    Run a solution's tests.
+.DESCRIPTION
+    Run a solution's tests.
+.PARAMETER Exercise
+    The slug of the exercise which tests to run.
+.PARAMETER InputDirectory
+    The directory in which the solution can be found.
+.PARAMETER OutputDirectory
+    The directory to which the results will be written.
+.EXAMPLE
+    The example below will run the tests of the two-fer solution in the "~/exercism/two-fer" directory
+    and write the results to the "~/exercism/results/" directory
+    PS C:\> ./run.ps1 two-fer ~/exercism/two-fer ~/exercism/results/
+#>
+
+param (
+    [Parameter(Position = 0, Mandatory = $true)]
+    [string]$Exercise,
+    
+    [Parameter(Position = 1, Mandatory = $true)]
+    [string]$InputDirectory,
+    
+    [Parameter(Position = 2, Mandatory = $true)]
+    [string]$OutputDirectory
+)
+
+$ErrorActionPreference = 'Stop'
+
+function Get-Exercise-Name {
+    (Get-Culture).TextInfo.ToTitleCase($Exercise).Replace("-", "")
+}
+
+$exerciseName = Get-Exercise-Name
+
+$testFile = Join-Path $InputDirectory "${exerciseName}Test.fs"
+$resultsTrxFile = Join-Path $InputDirectory "TestResults" "results.trx"
+$resultsJsonFile = Join-Path $OutputDirectory "results.json"
+
+filter Unskip-All-Tests {
+    $_ -Replace "Skip = ""Remove to run test""", ""
+}
+
+function Enable-All-Tests {
+    Get-Content $testFile | Unskip-All-Tests | Set-Content $testFile
+}
+
+filter Sanitize-Test-Output {
+    ($_ -Replace "\[$InputDirectory.*?\]", "").Trim()
+}
+
+function Run-All-Tests {
+    Remove-Item $resultsTrxFile -ErrorAction Ignore
+
+    $logger = "trx;LogFileName=results.trx"
+    dotnet test $InputDirectory --logger:$logger | Out-String | Sanitize-Test-Output
+}
+
+function Get-Tests-Ordered-By-Line-Number {
+    $testFileContents = Get-Content $testFile
+    $testMethodRegex = "let\s+````([\w ]+)````\s+\("
```

Comment:
I'm not entirely sure it works, but you may try something like this (a greedy match):

```suggestion
    $testMethodRegex = "let\s+````(.+?)````\s+\("
```

This version will also support using things like apostrophes and such.